src/test-utils/ui.js

```diff
@@ -0,0 +1,24 @@
+import { screen } from '@testing-library/react'
```

Comment:
I imagine that this file will be filled out more over time with many more React Testing Library helpers.